build.gradle.kts

```diff
@@ -29,6 +29,7 @@ dependencies {
     testImplementation("org.springframework.boot:spring-boot-starter-test") {
         exclude(group = "org.junit.vintage", module = "junit-vintage-engine")
     }
+    implementation("com.squareup.moshi:moshi-kotlin:1.9.2")
```

Comment:
What is the license for this dependency?

Reply:
It is Apache 2.0 as stated [here](https://github.com/square/moshi/#license).